src/components/NewGalaxyModal.js

```diff
@@ -83,23 +99,29 @@ export const NewGalaxyModal = _.flow(
       }],
       ['paused', () => {
         return h(Fragment, [
-          h(ButtonPrimary, { style: { marginRight: 'auto' }, onClick: () => setViewMode('deleteWarn') }, ['Delete']),
-          h(ButtonSecondary, { style: { marginRight: '1rem' }, onClick: resumeGalaxy }, ['Resume'])
+          deleteButton(),
+          resumeButton()
         ])
       }],
       [Utils.DEFAULT, () => !app ?
-        h(ButtonPrimary, { onClick: () => setViewMode('createWarn') }, ['Next']) :
+        h(ButtonPrimary, { disabled: false, onClick: () => setViewMode('createWarn') }, ['Next']) :
         Utils.switchCase(app.status,
           ['RUNNING', () => h(Fragment, [
-            deleteButton,
-            h(ButtonSecondary, { style: { marginRight: '1rem' }, onClick: () => { pauseGalaxy() } }, ['Pause']),
-            h(ButtonPrimary, { onClick: () => setViewMode('launchWarn') }, ['Launch Galaxy'])
+            deleteButton(),
+            pauseButton(),
+            h(ButtonPrimary, { disabled: false, onClick: () => setViewMode('launchWarn') }, ['Launch Galaxy'])
           ])],
           ['STOPPED', () => h(Fragment, [
-            deleteButton,
-            h(ButtonPrimary, { style: { marginRight: '1rem' }, onClick: () => { resumeGalaxy() } }, ['Resume'])
+            h(ButtonSecondary, { disabled: true, style: { marginRight: 'auto' }, tooltip: 'Cloud Compute must be resumed first.', onClick: () => setViewMode('deleteWarn') }, ['Delete']),
+            resumeButton()
           ])],
-          ['ERROR', () => deleteButton]
+          ['ERROR', () => deleteButton],
+          [Utils.DEFAULT, () => {
+            return h(Fragment, [
+              deleteButton(),
```

Comment:
I think it should be `deleteButton(true)` here -- otherwise it makes `stopping`, `starting` deletable which is not what we want.

Reply:
Example from local testing. The app now is `Starting` and the `DELETE` button is clickable:
![image](https://user-images.githubusercontent.com/5368863/112678928-ef13c200-8e41-11eb-9cc1-d579d3af8e81.png)